src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,8 +15,13 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL =
+            "https://github.com/AY2122S2-CS2103-F11-2/tp/blob/master/docs/UserGuide.md";
+    public static final String HELP_MESSAGE = "Available functions on this version: \n\n"
+            + "-add n/NAME id/STUDENT_ID c/COURSE \n-delete STUDENT_ID\n"
```

Comment:
Perhaps the `add` command should include all the parameters.

Reply:
WJ, could also add a spacing between the `-` and `TEXT` in your next code PR